apple/Sources/FerrostarCore/FerrostarCore.swift

```diff
@@ -81,6 +81,8 @@ public protocol FerrostarCoreDelegate: AnyObject {
     /// The observable state of the model (for easy binding in SwiftUI views).
     @Published public private(set) var state: NavigationState?
 
+    public let annotation: (any AnnotationPublishing)?
```

Comment:
This is a good discussion. I left it as annotation so you could access it like:

```swift
ferrostarCore.annotation.$speedLimit
```

We also provably want to use the annotation published class to transform the congestion and apply it to the route object in whole (and whatever else comes from annotation json). These two factors combined might push us to something entirely different for naming.

Reply:
Ooooh, that's a good point.... Yes, I think I see it now...

Here's my half-baked thoughts on it...

1. Leave it as-is prop name wise.
2. We can improve readability by renaming the `value` prop in the `AnnotationPublishing` protocol to `currentValue`.
3a. This opens the possibility for further extension of the protocol to get, say, _all_ values of an annotation (maybe chunked by step or something? Ideas welcome...) This would let us accommodate the cases like congestion annotations which really are designed to be overlaid over the full route.
3b. Alternate idea: if this doesn't work, we could rename the protocol to reflect that it's only concerned with the CURRENT value of an annotation (based on trip progress) and come up with a different way of accessing the "full-route" annotations.

I don't personally know enough to know which option is better. Let's discuss in tomorrow's meetings.